provision.sh

```diff
@@ -36,3 +42,16 @@ apt-get -y install vim
 ### Install screen
 apt-get -y install screen
 
+### Install Docker
+# https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-using-the-convenience-script
+curl -fsSL get.docker.com -o get-docker.sh
+sh get-docker.sh
+rm get-docker.sh
+
+### Setup Docker without sudo
+usermod -aG docker vcm
```

Comment:
should we have vcm here have the same check as above on line 27?

Reply:
Good catch. Please edit and merge.